LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -466,6 +466,7 @@ private void SendClassCDeviceMessage(IReceivedLoRaCloudToDeviceMessage cloudToDe
         {
             if (this.classCDeviceMessageSender != null)
             {
+                // Exception is logged as part of DefaultClassCDevicesMessageSender.SendAsync.
                 _ = Task.Run(() => this.classCDeviceMessageSender.SendAsync(cloudToDeviceMessage));
```

Comment:
Technically we are calling an interface here and we are relying on the implementation to do the logging. I would lift the exception handler and logging out of `DefaultClassCDevicesMessageSender.SendAsync` and put it here. After `UdpServer` is retired, this will be the only call site.

Reply:
Since that would mean that we refactor many tests and also change the signature of the method (it would not return a `Task<bool>` any longer): What do you think about renaming the method to `TrySendAsync`? That would indicate that it should always succeed, and it would leave the signature intact.

Reply:
Addressed with 1c55bfb109342723c3ac68b6a650843c8a14f7f1